product_variants_no_automatic_creation/models/pricelist.py

```diff
@@ -0,0 +1,160 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+# For copyright and license notices, see __openerp__.py file in root directory
+##############################################################################
+
+from openerp import models, fields, tools, api, exceptions, _
+
+
+class ProductPricelist(models.Model):
+    _inherit = 'product.pricelist'
+
+    @api.model
+    def _price_rule_get_multi(self, pricelist, products_by_qty_by_partner):
```

Comment:
:sweat: do not remember why... we've done modules for sale, purchase and mrp, I think they need this one, but I can't remember why...